client/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Dojo
+Copyright (c) Bibliotheca DAO
```

Comment:
@supreme2580 why did you change this copyright to Bibliotheca?

Please change to Realm of Ra instead